cmd/pkger/cmds/pack.go

```diff
@@ -147,6 +147,7 @@ func Package(info here.Info, out string, decls parser.Decls) error {
 		name = "main"
 	}
 
+	fmt.Fprintf(f, "// Code generated by pkger; DO NOT EDIT.\n\n", name)
```

Comment:
I just noticed that the `name` variable was accidentally left in after a copy paste from the line below.

It will cause `go vet -printf` to fail

```
go vet -printf
# github.com/markbates/pkger/cmd/pkger/cmds
./pack.go:150:13: Fprintf call has arguments but no formatting directives
```

Reply:
Would you mind another PR to fix? Thanks.

Reply:
Sure. See #70.